Nahokyun/boj/boj2666.java

```diff
@@ -0,0 +1,115 @@
+package boj;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.StringTokenizer;
+
+public class boj2666 {
+
+    private static int size;
+    static int max = Integer.MAX_VALUE;
+    private static boolean[] wall;
+    private static int[] seq;
+    private static int n;
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        n = Integer.parseInt(br.readLine());
+
+        wall = new boolean[n + 2];
+
+        StringTokenizer st = new StringTokenizer(br.readLine());
+
+        for (int i = 0; i < 2; i++) {
+            wall[Integer.parseInt(st.nextToken())] = true;
+        }
+        size = Integer.parseInt(br.readLine());
+
+        seq = new int[size + 1];
+        for (int i = 1; i <= size; i++) {
+            seq[i] = Integer.parseInt(br.readLine());
+        }
+        int leftdoor = -1;
+        int rightdoor = -1;
+        boolean flag = true;
+        for (int i = 1; i <= n; i++) {
+            if (wall[i] && flag) {
+                leftdoor = i;
+                flag = false;
+            } else if (wall[i]) {
+                rightdoor = i;
+            }
+        }
```

Comment:
leftdoor , rightdoor을 처음 입력받을 때 지정해주면 반복문을 돌지 않고도 초기 leftdoor, rightdoor을 찾을 수 있을 것 같습니다 😄
```suggestion
        leftdoor = Integer.parseInt(st.nextToken());
        rightdoor = Integer.parseInt(st.nextToken());
        
        wall[leftdoor] = true;
        wall[rightdoor] = true;
```

Reply:
앗 그러네요 개수도 두개로 정해져 있고 따로 입력도 받는데 쓸모없는 로직이 있었네요! 좋은 지적 감사합니다 : )